bug-1690385-pref-activity-stream-new-tab-default-content-release-85-87

```diff
@@ -0,0 +1,152 @@
+[experiment]
+end_date = '2020-04-13'
+
+segments = ["attributed", "non_attributed"]
+
+enrollment_period = 8
+
+reference_branch = "baseline-newtab-content"
+
+
+## data sources
+
+[data_sources]
+
+## data source: AS session pings data
+[data_sources.as_session_ping]
+from_expression = "(SELECT client_id, user_prefs, date(submission_timestamp) as submission_date FROM  `moz-fx-data-shared-prod.activity_stream.sessions`)"
+
+experiments_column_type = "native"
+
+
+
+
+## Segments
+
+[segments]
+
+[segments.attributed]
+select_expression = 'attr_source is not null'
+data_source = "npp"
+
+[segments.non_attributed]
+select_expression = 'attr_source is null'
+data_source = "npp"
+
+[segments.data_sources.npp]
```

Comment:
When are you expecting to receive a new_profile ping from these clients? If you think you'll always receive them on the same day as the enrollment ping, then this will work the way you're expecting. If you think you might have received a new_profile ping on a day _before_ the client enrolls, then we'll need to provide a `window_start` value (if the profiles might be just a day or two old) or do some aggregation aforethought (if profiles might be months old).

Reply:
got it. will assume majority of users report same day for simplicity right now, but will keep an eye out if it's worth windowing them for completeness.

Reply:
hmm, actually... considering my below comment, I think I actually want to be conservative here, and use the `window_start` value here... to not mislabel any users. 

is there any docs on how to specify a window start for a segements data source? I didn't catch it in the docs.

Reply:
The clearest explanation is https://mozilla.github.io/mozanalysis/api/segments.html#mozanalysis.segments.SegmentDataSource -- Jetstream accepts window_start and window_end arguments. It'd look like e.g. `window_start = -1`.

Reply:
sounds good. 

I added the `window_start` params into the `[segments.data_sources.npp]` data source section, based on the documentation (it seems to be defined where you define the data source). 

Please correct me if I'm wrong.